feat Remember last romm booked

JessicaBuenfilSolis opened the pull request.

JessicaBuenfilSolis committed:
feat Remember last romm booked

mishelashala commented on src/components/Modals/CreateMeeting/BookingForm.jsx:
```diff
@@ -85,6 +85,10 @@ const styles = theme => ({
   }
 });
 
+const setLastRoom = room => {
```
you're just forwarding props. no need for this function

mishelashala commented on src/components/Modals/CreateMeeting/BookingForm.jsx:
```diff
@@ -264,9 +268,11 @@ class BookingFormComponent extends React.Component {
           return this.saveBookingResponse(bookingInfo, isBookingEdition);
         }
         const bookingInfo = await this.doBookingCreation(booking);
+        setLastRoom(this.state.roomId);
```
use `storageService.setLastRoom`

mishelashala commented on src/pages/Login/Login.jsx:
```diff
@@ -18,6 +19,11 @@ const style = () => ({
 });
 
+const getLastRoom = () => {
+  const room = storageService.getLastRoom();
```
no need for assignment

```tsx
  return storageService.getLastRoom() || "1";
```

JessicaBuenfilSolis committed:
feat remember last booked room

attending changes requested
delete function setLastRoom
improve function getLastRoom
